src/renderer/services/binance/fees.ts

```diff
@@ -16,18 +17,21 @@ const { stream$: reloadFees$, trigger: reloadFees } = triggerStream()
  * Transaction fees
  */
 const fees$: (client: Client$) => FeesLD = (client$) =>
-  Rx.combineLatest([reloadFees$, client$]).pipe(
+  Rx.combineLatest([reloadFees$, client$.pipe(startWith(O.none as O.Option<Client>))]).pipe(
```

Comment:
Why do we need `startWith(O.none as O.Option<Client>)` here? Is not `O.none` the default value of any `Client$`?

Reply:
nice catch. seems this is an artifact from debugging